source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss

```diff
@@ -300,6 +304,36 @@ $font-family: 'Segoe UI', sans-serif;
                 outline: $validation-failed-outline;
             }
         }
+
+        &.ac-textInput-readWriteStyle,
+        &.ac-numberInput-readWriteStyle,
+        &.ac-timeInput-readWriteStyle,
+        &.ac-dateInput-readWriteStyle,
+        &.ac-choiceSetInput-compact-readWriteStyle {
+            border: $border-readWrite-style-onrender;
+            background-color: $background-color-readWrite-style-onrender;
+            &:hover {
+                border: $border-readWrite-style-onhover;
+            }
+        }
+
+        &.ac-choiceSetInput-compact-readWriteStyle {
+            appearance: none;
+        }
+    }
+
+    .ac-adaptiveCard:hover .ac-textInput-readWriteStyle,
+    .ac-adaptiveCard:hover .ac-numberInput-readWriteStyle,
+    .ac-adaptiveCard:hover .ac-timeInput-readWriteStyle,
+    .ac-adaptiveCard:hover .ac-dateInput-readWriteStyle,
+    .ac-adaptiveCard:hover .ac-choiceSetInput-compact-readWriteStyle {
+        border: $border-readWrite-style-oncard-hover;
```

Comment:
You already define a `:hover` style for the card, why not define it in such a way that it also updates the style of the card's input, instead of relying on JS events?

And again, don't use input-type-specific class names, use a separate `ac-inputStyle-readWrite` (I should say `ac-inputStyle-borderLess`) and use it in connjunction with `ac-input`.